rest/config_manager_test.go

```diff
@@ -51,3 +52,87 @@ func TestBootstrapConfig(t *testing.T) {
 	_, err = bootstrapContext.GetConfig(bucketName, configGroup1, db1Name, dbConfig1)
 	require.Error(t, err)
 }
+
+func TestComputeMetadataID(t *testing.T) {
+
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server - requires bootstrap support")
+	}
+
+	base.TestRequiresCollections(t)
+	// Start SG with no databases
+	config := BootstrapStartupConfigForTest(t)
+	ctx := base.TestCtx(t)
+	sc, err := SetupServerContext(ctx, &config, true)
+	require.NoError(t, err)
+	defer func() {
+		sc.Close(ctx)
+	}()
```

Comment:
I'm surprised that you can't use 

`NewRestTester(t, &RestTesterConfig{PersistentConfig: true})` for this. I don't have a problem with the approach you did but it's more wordy.